specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1,745 +1,886 @@
+                    "description": "The identity of SMS message receiver",
+                    "type": "string"
+                }
+            }
+        },
+        "AcsSmsDeliveryAttemptProperties": {
+            "description": "Schema for details of a delivery attempt",
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "description": "TimeStamp when delivery was attempted",
+                    "format": "date-time",
+                    "type": "string"
+                },
+                "segmentsSucceeded": {
+                    "description": "Number of segments that were successfully delivered",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "segmentsFailed": {
+                    "description": "Number of segments whose delivery failed",
+                    "type": "integer",
+                    "format": "int64"
+                }
+            }
+        },
+        "AcsRecordingStorageInfoProperties": {
+            "description": "Schema for all properties of Recording Storage Information.",
+            "type": "object",
+            "properties": {
+                "recordingChunks": {
+                    "description": "List of details of recording chunks information",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/AcsRecordingChunkInfoProperties"
+                    },
+                    "x-ms-identifiers": [
+                        "index"
+                    ]
+                }
+            }
+        },
+        "AcsRecordingChunkInfoProperties": {
+            "description": "Schema for all properties of  Recording Chunk Information.",
+            "type": "object",
+            "properties": {
+                "documentId": {
+                    "description": "The documentId of the recording chunk",
+                    "type": "string"
+                },
+                "index": {
+                    "description": "The index of the recording chunk",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "endReason": {
+                    "description": "The reason for ending the recording chunk",
+                    "type": "string"
+                },
+                "metadataLocation": {
+                    "description": "The location of the metadata for this chunk",
+                    "type": "string"
+                },
+                "contentLocation": {
+                    "description": "The location of the content for this chunk",
+                    "type": "string"
+                },
+                "deleteLocation": {
+                    "description": "The location to delete all chunk storage",
+                    "type": "string"
+                }
+            }
+        },
+        "AcsCallGroupProperties": {
```

Comment:
Are we expecting to add more properties here in the future?

Reply:
Not sure about future, but group {id : <groupId>} is the structure currently.